From 80120012e9072acb7433db1c04891a61295ab469 Mon Sep 17 00:00:00 2001
From: Jenkins <dev-kitchen@cloudera.com>
Date: Fri, 12 Apr 2019 04:17:01 -0700
Subject: [PATCH 489/489] Branching for 5.16.2 on Fri Apr 12 04:16:37 PDT 2019

JOB_NAME : 'Cut-Release-Branches'
BUILD_NUMBER : '579'
CODE_BRANCH : ''
OLD_CDH_BRANCH : 'cdh5_5.16.x'

Pushed to remote apache	git://github.infra.cloudera.com/Mirrors/apache-hbase.git (push)
origin	git://github.infra.cloudera.com/CDH/hbase.git (push)
---
 hbase-annotations/pom.xml                          |    2 +-
 hbase-assembly/pom.xml                             |    2 +-
 hbase-checkstyle/pom.xml                           |    4 ++--
 hbase-client/pom.xml                               |    2 +-
 hbase-common/pom.xml                               |    2 +-
 hbase-examples/pom.xml                             |    2 +-
 hbase-external-blockcache/pom.xml                  |    2 +-
 hbase-hadoop-compat/pom.xml                        |    2 +-
 hbase-hadoop2-compat/pom.xml                       |    2 +-
 hbase-it/pom.xml                                   |    2 +-
 hbase-prefix-tree/pom.xml                          |    2 +-
 hbase-procedure/pom.xml                            |    2 +-
 hbase-protocol/pom.xml                             |    2 +-
 hbase-resource-bundle/pom.xml                      |    2 +-
 hbase-rest/pom.xml                                 |    2 +-
 hbase-rsgroup/pom.xml                              |    2 +-
 hbase-server/pom.xml                               |    2 +-
 hbase-shaded/hbase-shaded-check-invariants/pom.xml |    2 +-
 hbase-shaded/hbase-shaded-client/pom.xml           |    2 +-
 hbase-shaded/hbase-shaded-server/pom.xml           |    2 +-
 hbase-shaded/pom.xml                               |    2 +-
 hbase-shell/pom.xml                                |    2 +-
 hbase-spark/pom.xml                                |    2 +-
 hbase-testing-util/pom.xml                         |    2 +-
 hbase-thrift/pom.xml                               |    2 +-
 pom.xml                                            |    4 ++--
 26 files changed, 28 insertions(+), 28 deletions(-)

diff --git a/hbase-annotations/pom.xml b/hbase-annotations/pom.xml
index 0be9bab..a58a40f 100644
--- a/hbase-annotations/pom.xml
+++ b/hbase-annotations/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.2-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.2</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-assembly/pom.xml b/hbase-assembly/pom.xml
index 72fcb05..0496f7a 100644
--- a/hbase-assembly/pom.xml
+++ b/hbase-assembly/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.2-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.2</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-assembly</artifactId>
diff --git a/hbase-checkstyle/pom.xml b/hbase-checkstyle/pom.xml
index 3efd249..aacd92d 100644
--- a/hbase-checkstyle/pom.xml
+++ b/hbase-checkstyle/pom.xml
@@ -24,14 +24,14 @@
 <modelVersion>4.0.0</modelVersion>
 <groupId>org.apache.hbase</groupId>
 <artifactId>hbase-checkstyle</artifactId>
-<version>1.2.0-cdh5.16.2-SNAPSHOT</version>
+<version>1.2.0-cdh5.16.2</version>
 <name>Apache HBase - Checkstyle</name>
 <description>Module to hold Checkstyle properties for HBase.</description>
 
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.2-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.2</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-client/pom.xml b/hbase-client/pom.xml
index 1de731b..e012e71 100644
--- a/hbase-client/pom.xml
+++ b/hbase-client/pom.xml
@@ -24,7 +24,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.2-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.2</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-common/pom.xml b/hbase-common/pom.xml
index 609a679..1baaa1f 100644
--- a/hbase-common/pom.xml
+++ b/hbase-common/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.2-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.2</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-examples/pom.xml b/hbase-examples/pom.xml
index 5764985..30b23c7 100644
--- a/hbase-examples/pom.xml
+++ b/hbase-examples/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.2-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.2</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-examples</artifactId>
diff --git a/hbase-external-blockcache/pom.xml b/hbase-external-blockcache/pom.xml
index e66ef1f..72ae014 100644
--- a/hbase-external-blockcache/pom.xml
+++ b/hbase-external-blockcache/pom.xml
@@ -25,7 +25,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.2-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.2</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-external-blockcache</artifactId>
diff --git a/hbase-hadoop-compat/pom.xml b/hbase-hadoop-compat/pom.xml
index 14fbe29..cbaca3e 100644
--- a/hbase-hadoop-compat/pom.xml
+++ b/hbase-hadoop-compat/pom.xml
@@ -23,7 +23,7 @@
     <parent>
         <artifactId>hbase</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.16.2-SNAPSHOT</version>
+        <version>1.2.0-cdh5.16.2</version>
         <relativePath>..</relativePath>
     </parent>
 
diff --git a/hbase-hadoop2-compat/pom.xml b/hbase-hadoop2-compat/pom.xml
index f3f5428..6087701 100644
--- a/hbase-hadoop2-compat/pom.xml
+++ b/hbase-hadoop2-compat/pom.xml
@@ -21,7 +21,7 @@ limitations under the License.
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.2-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.2</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-it/pom.xml b/hbase-it/pom.xml
index 944db0e..d1c4cdc 100644
--- a/hbase-it/pom.xml
+++ b/hbase-it/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.2-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.2</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-prefix-tree/pom.xml b/hbase-prefix-tree/pom.xml
index 1ecac6c..bd0eeba 100644
--- a/hbase-prefix-tree/pom.xml
+++ b/hbase-prefix-tree/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.2-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.2</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-procedure/pom.xml b/hbase-procedure/pom.xml
index 8079573..a7f6c70 100644
--- a/hbase-procedure/pom.xml
+++ b/hbase-procedure/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.2-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.2</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-protocol/pom.xml b/hbase-protocol/pom.xml
index 321e35f..23736e2 100644
--- a/hbase-protocol/pom.xml
+++ b/hbase-protocol/pom.xml
@@ -23,7 +23,7 @@
     <parent>
         <artifactId>hbase</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.16.2-SNAPSHOT</version>
+        <version>1.2.0-cdh5.16.2</version>
         <relativePath>..</relativePath>
     </parent>
 
diff --git a/hbase-resource-bundle/pom.xml b/hbase-resource-bundle/pom.xml
index 07a1ae2..fc70bc9 100644
--- a/hbase-resource-bundle/pom.xml
+++ b/hbase-resource-bundle/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.2-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.2</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-rest/pom.xml b/hbase-rest/pom.xml
index 7372b92..08d7286 100644
--- a/hbase-rest/pom.xml
+++ b/hbase-rest/pom.xml
@@ -25,7 +25,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.2-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.2</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-rest</artifactId>
diff --git a/hbase-rsgroup/pom.xml b/hbase-rsgroup/pom.xml
index 191bd7f..2d6f1e3 100644
--- a/hbase-rsgroup/pom.xml
+++ b/hbase-rsgroup/pom.xml
@@ -24,7 +24,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.2-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.2</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-server/pom.xml b/hbase-server/pom.xml
index de64d12..3aaf528 100644
--- a/hbase-server/pom.xml
+++ b/hbase-server/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.2-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.2</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-server</artifactId>
diff --git a/hbase-shaded/hbase-shaded-check-invariants/pom.xml b/hbase-shaded/hbase-shaded-check-invariants/pom.xml
index 863b865..11695fa 100644
--- a/hbase-shaded/hbase-shaded-check-invariants/pom.xml
+++ b/hbase-shaded/hbase-shaded-check-invariants/pom.xml
@@ -16,7 +16,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.2-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.2</version>
     <relativePath>../..</relativePath>
   </parent>
   <artifactId>hbase-shaded-check-invariants</artifactId>
diff --git a/hbase-shaded/hbase-shaded-client/pom.xml b/hbase-shaded/hbase-shaded-client/pom.xml
index 84433ed..5bd8009 100644
--- a/hbase-shaded/hbase-shaded-client/pom.xml
+++ b/hbase-shaded/hbase-shaded-client/pom.xml
@@ -24,7 +24,7 @@
     <parent>
         <artifactId>hbase-shaded</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.16.2-SNAPSHOT</version>
+        <version>1.2.0-cdh5.16.2</version>
         <relativePath>..</relativePath>
     </parent>
     <artifactId>hbase-shaded-client</artifactId>
diff --git a/hbase-shaded/hbase-shaded-server/pom.xml b/hbase-shaded/hbase-shaded-server/pom.xml
index 1d87123..6cfa7d5 100644
--- a/hbase-shaded/hbase-shaded-server/pom.xml
+++ b/hbase-shaded/hbase-shaded-server/pom.xml
@@ -24,7 +24,7 @@
     <parent>
         <artifactId>hbase-shaded</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.16.2-SNAPSHOT</version>
+        <version>1.2.0-cdh5.16.2</version>
         <relativePath>..</relativePath>
     </parent>
     <artifactId>hbase-shaded-server</artifactId>
diff --git a/hbase-shaded/pom.xml b/hbase-shaded/pom.xml
index 29f3a3b..2024a48 100644
--- a/hbase-shaded/pom.xml
+++ b/hbase-shaded/pom.xml
@@ -23,7 +23,7 @@
     <parent>
         <artifactId>hbase</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.16.2-SNAPSHOT</version>
+        <version>1.2.0-cdh5.16.2</version>
         <relativePath>..</relativePath>
     </parent>
     <artifactId>hbase-shaded</artifactId>
diff --git a/hbase-shell/pom.xml b/hbase-shell/pom.xml
index 257ffe7..260863b 100644
--- a/hbase-shell/pom.xml
+++ b/hbase-shell/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.2-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.2</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-shell</artifactId>
diff --git a/hbase-spark/pom.xml b/hbase-spark/pom.xml
index 837e807..0f7c36d 100644
--- a/hbase-spark/pom.xml
+++ b/hbase-spark/pom.xml
@@ -30,7 +30,7 @@
     <parent>
         <artifactId>hbase</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.16.2-SNAPSHOT</version>
+        <version>1.2.0-cdh5.16.2</version>
         <relativePath>..</relativePath>
     </parent>
     <artifactId>hbase-spark</artifactId>
diff --git a/hbase-testing-util/pom.xml b/hbase-testing-util/pom.xml
index cedb57d..8ddbf08 100644
--- a/hbase-testing-util/pom.xml
+++ b/hbase-testing-util/pom.xml
@@ -23,7 +23,7 @@
     <parent>
         <artifactId>hbase</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.16.2-SNAPSHOT</version>
+        <version>1.2.0-cdh5.16.2</version>
         <relativePath>..</relativePath>
     </parent>
     <artifactId>hbase-testing-util</artifactId>
diff --git a/hbase-thrift/pom.xml b/hbase-thrift/pom.xml
index fcfdfb7..ea8c934 100644
--- a/hbase-thrift/pom.xml
+++ b/hbase-thrift/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.2-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.2</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-thrift</artifactId>
diff --git a/pom.xml b/pom.xml
index 1a55e33..8b1d563 100644
--- a/pom.xml
+++ b/pom.xml
@@ -32,14 +32,14 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>cdh-root</artifactId>
-    <version>5.16.2-SNAPSHOT</version>
+    <version>5.16.2</version>
     <relativePath/>
     <!-- no parent resolution -->
   </parent>
   <groupId>org.apache.hbase</groupId>
   <artifactId>hbase</artifactId>
   <packaging>pom</packaging>
-  <version>1.2.0-cdh5.16.2-SNAPSHOT</version>
+  <version>1.2.0-cdh5.16.2</version>
   <name>Apache HBase</name>
   <description>
     Apache HBase is the Hadoop database. Use it when you need
-- 
1.7.9.5

